engine/src/main/java/org/hibernate/validator/internal/util/logging/Log.java

```diff
@@ -228,10 +246,10 @@ UnexpectedTypeException getNoValidatorFoundForTypeException(String constraintTyp
 	@Message(id = 63, value = "Unable to instantiate %s.")
 	ValidationException getUnableToInstantiateException(String className, @Cause Exception e);
```

Comment:
IIRC, the error is called when it's unable to instantiate the class. Thus we can't have a class parameter :).